### PR TITLE
Add OnThePixel.net

### PR DESCRIPTION
**Pull request by Paranoia8972**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/onthepixel/manifest.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "server_name": "onthepixel",
+  "nice_name": "OnThePixel Network",
+  "direct_ip": "play.onthepixel.net",
+  "server_wildcards": [
+    "%.onthepixel.net"
+  ],
+  "social": {
+    "web": "https://onthepixel.net",
```
Comment (JakobBruening) [inline minecraft_servers/onthepixel/manifest.json:9]: Not available.

File: minecraft_servers/onthepixel/manifest.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      "command": "/bw"
+    },
+    "buildffa":{
+      "name": "BuildFFA",
+      "command": "/bffa"
+    }
+  },
+  "discord": {
+    "server_id": 1051287687233015841,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/onthepixel/manifest.json:30]: Only available for partners. Please use `false`, as this is the default value.

File: minecraft_servers/onthepixel/manifest.json (hunk 1 above)
Comment (Paranoia8972) [reply minecraft_servers/onthepixel/manifest.json:9]: Hi! Thanks for your review! Is it a problem if I publish the website in one or two days when I’ve finished it?

**Comment by Paranoia8972** [comment]
Website is up and running! Hopefully ready to pull. Thanks for your review.

**Pull request merged**